packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -276,6 +373,35 @@ export abstract class BaseService extends Resource
     };
   }
 
+  /**
+   * Use this function to create all load balancer targets to be registered in this service, add them to
+   * target groups, and attach target groups to listeners accordingly.
+   *
+   * @example
+   *
+   * service.registerLoadBalancerTargets(
+   *   {
+   *     containerTarget: {
+   *       containerName: 'web',
+   *       containerPort: 80,
+   *     },
+   *     targetGroupId: 'ECS',
+   *     listener: ecs.ListenerConfig.applicationListener(listener, {
+   *       protocol: elbv2.ApplicationProtocol.HTTPS
+   *     }),
+   *   },
+   * )
```

Comment:
You should add this to the README, not sure if it makes sense to add it here.

Reply:
Do we have a precedent for having examples in jsdoc strings versus in README?